src/WebApp/appsettings.json

```diff
@@ -1,23 +1,23 @@
 {
-  "ConnectionStrings": {
-    "DefaultConnection": "--placeholder--"
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+    "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Username=codidact-core;Database=codidact-core"
```

Comment:
Here and in line 17, this PR changed a `~~placeholder~~` value to a concrete value. Is that intentional.

Reply:
This is a relic of me working on rewriting the documentation. I'll put it back for now.

Reply:
17c0831b58236f1d76d6da52eb95928bb7aeec60